<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>988</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="acc_edit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>551</width>
      <height>161</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="del_acc">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>90</y>
       <width>121</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete Account</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_1">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>130</y>
       <width>115</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Clerk</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>130</y>
       <width>115</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>User</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_3">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>130</y>
       <width>115</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Delivery</string>
     </property>
    </widget>
    <widget class="QPushButton" name="add_acc">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>40</y>
       <width>121</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Account</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_username">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>90</y>
       <width>220</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_psw">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>50</y>
       <width>220</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_username">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>299</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>User Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_psw">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>299</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="back">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>105</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Go back</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="acc_info">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>711</width>
      <height>391</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="view_compl">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>180</y>
       <width>131</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>View Complaints</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>0</y>
       <width>561</width>
       <height>301</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="view_users">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>111</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>View Users</string>
     </property>
    </widget>
    <widget class="QPushButton" name="view_clerks">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>111</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>View Clerks</string>
     </property>
    </widget>
    <widget class="QPushButton" name="view_avoid">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>140</y>
       <width>131</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>View Avoid List</string>
     </property>
    </widget>
    <widget class="QPushButton" name="view_Supliers">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>121</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>View Supliers</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="edit_acc">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>30</y>
      <width>105</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit Account</string>
    </property>
   </widget>
   <widget class="QPushButton" name="uptate_compl">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>70</y>
      <width>151</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Update Complaints</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>988</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>edit_acc</sender>
   <signal>clicked()</signal>
   <receiver>acc_info</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>577</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edit_acc</sender>
   <signal>clicked()</signal>
   <receiver>acc_edit</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>view_users</sender>
   <signal>clicked()</signal>
   <receiver>acc_edit</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>back</sender>
   <signal>clicked()</signal>
   <receiver>acc_info</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>282</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>480</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
